charts/base-cluster/templates/weave/gitops-dashboard.yaml

```diff
@@ -0,0 +1,56 @@
+{{- if .Values.weaveworks.enabled }}
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: weave
+  labels: {{- include "common.labels.standard" $ | nindent 4 }}
+---
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  annotations:
+    metadata.weave.works/description: This is the Weave GitOps Dashboard.  It provides
+      a simple way to get insights into your GitOps workloads.
+  name: gitops-dashboard
+  namespace: weave
+spec:
+  chart:
+    spec:
+      chart: weave-gitops
+      sourceRef:
+        kind: HelmRepository
+        name: weaveworks
+        namespace: flux-system
+  interval: 5m
+  values:
+    replicaCount: 1
+    {{- if and .Values.weaveworks.adminUser .Values.weaveworks.adminUser.create }}
+    adminUser:
+      create: true
+      username: {{ .Values.weaveworks.adminUser.username }}
+      createSecret: true
+      {{- if .Values.weaveworks.adminUser.password }}
+      {{- $password := split ":" (htpasswd .Values.weaveworks.adminUser.username .Values.weaveworks.adminUser.password) }}
```

Comment:
Shouldn't this be bcrypt? 🤔

Reply:
replaced. Looked up the function in the helm docu.